src/cli/cli_console.hpp

```diff
@@ -34,9 +34,10 @@
 #ifndef CLI_CONSOLE_HPP_
 #define CLI_CONSOLE_HPP_
 
-#include "cli_config.h"
 #include "openthread-core-config.h"
 
+#include "cli_config.h"
```

Comment:
should `#include "openthread-core-config.h"` be put into `cli_config.h`?

Reply:
Updated